src/AccessibilityInsights.SharedUxTests/Settings/ConfigurationModelTests.cs

```diff
@@ -4,9 +4,9 @@
 using AccessibilityInsights.Desktop.UIAutomation;
 using AccessibilityInsights.DesktopUI.Enums;
 using AccessibilityInsights.RuleSelection;
-using AccessibilityInsights.SetupLibrary;
 using AccessibilityInsights.SharedUx.Enums;
 using AccessibilityInsights.SharedUx.Settings;
+using AccessibilityInsights.SharedUx.Utilities;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using System;
 using System.Globalization;
```

Comment:
>35 [](start = 28, length = 2)

My merge to master will cause a merge conflict here

Reply:
thanks for the heads up, I have updated it